src/panels/AuditLogsFilterPanel/Fields/Fields.tsx

```diff
@@ -24,13 +25,15 @@ export enum Label {
   VERSION = "Version",
 }
 
+export const DATETIME_LOCAL = "yyyy-MM-dd'T'hh:mm";
```

Comment:
I think we need `HH` instead of `hh` here. The error that we were facing on Friday, where we could not insert the current time, most likely came from here, as `hh` represents the hour in 12-hour clock format instead of 24-hour clock like `HH` does.

Reply:
Ah thankyou!